qa/pull-tester/rpc-tests.py

```diff
@@ -478,9 +479,21 @@ def comms(timeout):
                     stdout = log_stdout.read()
                     stderr = log_stderr.read()
                     passed = stderr == "" and proc.returncode == 0
+
+                    # This is a list of expected messages on stderr. If they appear, they do not
+                    # necessarily indicate final failure of a test.
+                    stderr_filtered = stderr.replace("Error: Unable to start HTTP server. See debug log for details.", "")
```

Comment:
I really do like code changes made in this PR. It makes the QA py suite more robust/resilient to Travis platform weirdness. The only thing that I found a little bit fragile in the PR is this part, cause we need to keep in sync the py test w/ changes that could me made to bitcoind/qt code. E.g.:

Reply:
Excellent point regarding the locale and translations!

To be honest, I didn't like the string comparison that much either but found no quick solution (and didn't think about the locale issue).

One thing one could do is to maybe prefix fatal error messages of bitcoind with some kind of error message index and create a global table of them?

I imagine bitcoind could fail with something like:

[ERR#1234] Unable to start HTTP server. See debug log for details.

and then one would just create a regexp or string match for the first [ERR #1234] part. What do you think?

Reply:
Think this is a fair approach even if we won't be able to achieve the use of this set of error codes on all Cash implelemtation.